### PR TITLE
MDL-73699 qbank_editquestion: Question status UI/UX update

### PR DESCRIPTION
**Pull request by marcghaly**
Hi @safatshahin, here is the commit for the ui status change,
cheers !

### CODE REVIEW

**Comment by marcghaly** [comment]
Hi @safatshahin, this branch is rebased over latest `master_MDL-71696-versioning-integration`,
cheers!

File: question/bank/editquestion/templates/question_status_dropdown.mustache [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    Example context (json):
+    {
+        "ready" : "Ready",
+        "draft" : "Draft",
+        "selectedready" : "true",
+        "questionid" : "12"
+    }
+}}
+<select id="question_status_dropdown-{{questionid}}" class="custom-select" name="question_status_dropdown">
```
Comment (safatshahin) [inline question/bank/editquestion/templates/question_status_dropdown.mustache:28]: please remove the class

File: question/bank/editquestion/templates/question_status_dropdown.mustache [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template qbank_editquestion/question_status_dropdown
+
+    Example context (json):
+    {
```
[the remark this answers is not in the document]
Comment (marcghaly) [reply question/bank/editquestion/templates/question_status_dropdown.mustache:21]: Done.

File: question/bank/editquestion/templates/question_status_dropdown.mustache [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template qbank_editquestion/question_status_dropdown
+
```
[the remark this answers is not in the document]
Comment (marcghaly) [reply question/bank/editquestion/templates/question_status_dropdown.mustache:19]: Done.

File: question/bank/editquestion/templates/question_status_dropdown.mustache (hunk 1 above)
Comment (marcghaly) [reply question/bank/editquestion/templates/question_status_dropdown.mustache]: Removed and readded `custom-select`.

**Comment by safatshahin** [comment]
pushed for peer review to the ui/ux team, thanks for your contribution.
cheers

**Pull request closed without merging**